account-kit/react/src/components/auth/card/steps.tsx

```diff
@@ -14,11 +14,11 @@ export const Step = (props: AuthCardProps) => {
     case "email_verify":
     case "passkey_verify":
     case "email_completing":
-      return <LoadingAuth context={authStep} />;
+      return <LoadingAuth config={props} context={authStep} />;
```

Comment:
I imagine we'll be moving this out into context once we put the `uiConfig` into a `/react` specific `createConfig` method right?

Reply:
definitely, didn't want to put more in here since this is already a large-ish PR but aligned 🫡